addons/point_of_sale/static/src/app/screens/product_screen/action_pad/action_pad.js

```diff
@@ -27,6 +27,9 @@ export class ActionpadWidget extends Component {
     get isLongName() {
         return this.props.partner && this.props.partner.name.length > 10;
     }
+    get highlightPay() {
+        return this.pos.get_order()?.lines?.length;
+    }
```

Comment:
```suggestion
```
Unused. Maybe unnecessary because the issue about "Payment" and "Order" buttons being the same color is already fixed in master.